pkg/kubelet/certificate/bootstrap/bootstrap.go

```diff
@@ -62,7 +62,11 @@ func LoadClientCert(kubeconfigPath string, bootstrapPath string, certDir string,
 	if err != nil {
 		return fmt.Errorf("unable to load bootstrap kubeconfig: %v", err)
 	}
-	_, err = kubernetes.NewForConfig(bootstrapClientConfig)
+	client, err := kubernetes.NewForConfig(bootstrapClientConfig)
+	if err != nil {
+		return err
+	}
+	_, err = client.ServerVersion()
```

Comment:
as commented in https://github.com/kubernetes/kubernetes/pull/50298, I don't think these extra checks are necessary, we'll already return errors when we try to use the API if we can't reach it